tests/database/mysql/mySqlDatabase.test.js

```diff
@@ -8,6 +8,8 @@ import dbCreate from "zoapp-core/database";
 
 import { descriptor, dbConfig } from "../../test-config";
 
+jest.setTimeout(50000);
```

Comment:
I dont think we should be doing this, but maybe you could explain what happens?

Reply:
time to time it could took some delay to init/access DB

Reply:
I see, but I dont think we should be doing this. This is not really a best practice.

Reply:
Ok i removed. I think all is ok now ;-)